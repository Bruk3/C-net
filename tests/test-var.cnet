int a; 

// test 

int y = 7; 

