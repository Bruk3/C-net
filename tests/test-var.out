INT
ID(a)
SEMI
INT
ID(y)
ASSIGN
INTLIT(7)
SEMI
EOF